src/main/rx/rx.c

```diff
@@ -449,6 +449,14 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTime)
             rxIsInFailsafeMode = (frameStatus & RX_FRAME_FAILSAFE) != 0;
             rxSignalReceived = !rxIsInFailsafeMode;
             needRxSignalBefore = currentTimeUs + needRxSignalMaxDelayUs;
+
+            if (frameStatus & RX_FRAME_DROPPED) {
+            	// No (0%) signal
+            	setRssiUnfiltered(0, RSSI_SOURCE_NONE);
```

Comment:
I've had a think about this, and using `RSSI_SOURCE_NONE` here is wrong. This shouldn't be a 'we'll do it while nothing else sends RSSI, and then we'll stop' thing, but it should be something that the user explicitly enables (and as part of that acknowledges that this does not act in the same more or less linear fashion that 'normal' RSSI does.
Please add a boolean parameter `rssi_from_frame_errors` and  `RSSI_SOURCE_FRAME_ERRORS` (or use a better name if you come up with one), and use this to set up, and then use this source.

Reply:
Will do. I just used an option that the GUI gave me to work with.

Reply:
sbusFrameStatus() in sbus.c and sbusChannelsDecode() ins bus_channels.c both process the SBUS_FLAG_FAILSAFE_ACTIVE flag so I fear we have some duplication here.

Reply:
I've removed the flag processing from sbusFrameStatus() as it was superfluous. rx.h defines RSSI_SOURCE_RX_PROTOCOL which I think is fit for purpose, so I don't think we need to add RSSI_SOURCE_FRAME_ERRORS.

It's worth noting that the fport driver uses this, and contains the following code in fportRxInit()

```
        if (rssiSource == RSSI_SOURCE_NONE) {
            rssiSource = RSSI_SOURCE_RX_PROTOCOL;
        }
```

Perhaps the Config GUI should add an option to set the RSSI source to be the Rx protocol